paasta_tools/cli/cmds/validate.py

```diff
@@ -317,13 +330,33 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
-def validate_tron(service_path):
+def validate_tron(service_path: str, verbose: bool = False) -> bool:
     soa_dir, service = path_to_soa_dir_service(service_path)
     returncode = True
 
     for cluster in list_tron_clusters(service, soa_dir):
         if not validate_tron_namespace(service, cluster, soa_dir):
             returncode = False
+        elif verbose:
+            # service config has been validated and cron schedules are safe to parse
+            service_config = load_tron_service_config(service, cluster)
+            for config in service_config:
+                schedule = config.get_schedule()
+
+                if isinstance(schedule, str) and schedule.startswith("cron"):
```

Comment:
i think the last thing we'll want to handle is the dict form of a cron schedule (which i didn't know was a thing until today) - maybe we add a helper for figuring out if a schedule is cron-based and another for actually getting the cron expression?